config/prod.exs

```diff
@@ -143,17 +143,22 @@ config :archethic, Archethic.Mining.PendingTransactionValidation,
         :software
     end)
 
+# -----Start-of-Networking-prod-configs-----
+
+config :archethic, Archethic.Networking,
+  validate_node_ip: System.get_env("ARCHETHIC_NODE_IP_VALIDATION", "true") == "true"
+
 config :archethic,
        Archethic.Networking.IPLookup,
        (case(System.get_env("ARCHETHIC_NETWORKING_IMPL", "NAT") |> String.upcase()) do
           "NAT" ->
-            Archethic.Networking.IPLookup.NAT
+            Archethic.Networking.IPLookup.NATDiscovery
 
           "STATIC" ->
             Archethic.Networking.IPLookup.Static
 
-          "IPFY" ->
-            Archethic.Networking.IPLookup.IPIFY
+          "IPIFY" ->
```

Comment:
I think you can change it to RemoteDiscovery, we will update the snap conf as well

Reply:
done